packages/frontend/src/redux/actions/account.js

```diff
@@ -510,8 +545,8 @@ export const finishAccountSetup = () => async (dispatch, getState) => {
     if (promptTwoFactor) {
         dispatch(redirectTo('/enable-two-factor', { globalAlertPreventClear: true }));
     } else {
-        if (url?.redirectUrl) {
-            window.location = `${url.redirectUrl}?accountId=${accountId}`;
+        if (!!redirectUrl) {
```

Comment:
The `!!` can be omitted here for better consistency with other checks in this module.

Reply:
Done.